src/include/iscsi-lio-server/widgets.rb

```diff
@@ -1445,8 +1448,24 @@ def handleClient(key, event)
 
     def validateClient(key, event)
       event = deep_copy(event)
-      ret = true
-      ret
+      clients = Convert.convert(UI.QueryWidget(:clnt_table, :Items),
+                                :from => "any",
+                                :to   => "list <term>"
+                                )
+      if clients.empty?
+        return false if !Popup.AnyQuestion(
```

Comment:
Question is: do you want to continue without client list - it's not easy to decide what't positive here. User might want to create targets without client access (add clients later) or expect to have client access now (which requires client name).